charts/osm/values.yaml

```diff
@@ -6,7 +6,7 @@ OpenServiceMesh:
   image:
     registry: openservicemesh
     pullPolicy: IfNotPresent
-    tag: v0.3.0
+    tag: v0.4.0
```

Comment:
Can you pls confirm the image tag and the chart version and appVersion are the only updates in the osm chart going from https://github.com/openservicemesh/osm/releases/tag/v0.3.0 to https://github.com/openservicemesh/osm/releases/tag/v0.4.0

Reply:
The previous version of the chart that I had was from main, not the v0.3.0 release. So these were the only differences from the osm chart in main.

Reply:
Got it. Let's always use a released version of osm from now on.